src/NUnitEngine/nunit.engine/Runners/ProcessRunner.cs

```diff
@@ -232,7 +232,7 @@ TestEngineResult CreateFailedResult(Exception e)
             XmlHelper.AddAttribute(suite, "fullname", TestPackage.FullName);
             XmlHelper.AddAttribute(suite, "runstate", "NotRunnable");
             XmlHelper.AddAttribute(suite, "testcasecount", "1");
-            XmlHelper.AddAttribute(suite, "result", "Failed");
+            XmlHelper.AddAttribute(suite, "result", "UnexpectedError");
```

Comment:
The three attributes result, label and site (not used here) map to the three fields in a ResultState object. The result attribute represents the value of TestStatus, which only has four values. We consider those to be the four fundamentally different outcomes of a test: Passed, Failed, Inconclusive and Skipped. It would be best not to add another one without a darned good reason, and it should really be fundamentally different if we do.

The label in a ResultState is just a string and is intended to be used to create new sub-results. You could do that.

That said, it's possible that you might just use Failed:Error rather than inventing anything new. It depends whether an assembly ever has a Failed:Error state for any other reason. For example, if I remember right, we get a Failed:Invalid if the assembly can not be found.

BTW, we use the ResultState class in the framework but we have not translated the attributes back to an object for the console because it didn't seem necessary so far. YAGNI. OTOH, in the Gui, I think it will be reconstituted.